multinet/api.py

```diff
@@ -41,14 +95,32 @@ def get_workspaces() -> Any:
     return stream
 
 
-@bp.route("/workspaces/<workspace>", methods=["GET"])
-@require_reader
-@swag_from("swagger/workspace.yaml")
-def get_workspace(workspace: str) -> Any:
-    """Retrieve a single workspace."""
+@bp.route("/workspaces/<workspace>/permissions", methods=["GET"])
+@require_maintainer
+@swag_from("swagger/get_workspace_permissions.yaml")
+def get_workspace_permissions(workspace: str) -> Any:
+    """Retrieve the permissions of a workspace."""
     metadata = db.get_workspace_metadata(workspace)
 
-    return {"name": metadata["name"], "permissions": metadata["permissions"]}
+    return _permissions_id_to_user(metadata["permissions"])
```

Comment:
This seems to be the only call to `_permissions_id_to_user`. Why not inline the function here?

Reply:
I just preferred to keep those temporary functions clearly separated and marked from the rest of the api. Since eventually, we'll remove those functions, but the api endpoint will remain.